kurl_util/cmd/pvmigrate/main.go

```diff
@@ -37,31 +42,60 @@ func main() {
 	flag.BoolVar(&opts.VerboseCopy, "verbose-copy", false, "show output from the rsync command used to copy data between PVCs")
 	flag.BoolVar(&opts.SkipSourceValidation, "skip-source-validation", false, "migrate from PVCs using a particular StorageClass name, even if that StorageClass does not exist")
 	flag.BoolVar(&skipFreeSpaceCheck, "skip-free-space-check", false, "skips the check for storage free space prior to running the migrations")
+	flag.BoolVar(&printVersion, "version", false, "Print the version of the client")
+	flag.IntVar(&podReadyTimeout, "pod-ready-timeout", 60, "length of time to wait (in seconds) for volume validation pod(s) to go into Ready phase")
+	flag.IntVar(&deletePVTimeout, "delete-pv-timeout", 300, "length of time to wait (in seconds) for backing PV to be removed when temporary PVC is deleted")
+	flag.BoolVar(&skipPreflightValidation, "skip-preflight-validation", false, "skips pre-migration validation")
+	flag.BoolVar(&preflightValidationOnly, "preflight-validation-only", false, "skip the migration and run preflight validation only")
+
 	flag.Parse()
 
-	logger := log.New(os.Stdout, "", 0)
+	fmt.Printf("Running pvmigrate build:\n")
```

Comment:
This will get output as part of [these](https://github.com/replicatedhq/kURL/pull/3767/files#diff-a639b8e65bf2b4327bfb8a263798cad88f2c558c76e1dda96169edb3b7bcf4eeR85-R86) log lines. This should go to stderr using the logger.

Reply:
why do you think the version info belongs in stdout?

Reply:
i think it belongs in stderr but you have it going to stdout. in fact i would say it belongs in stdout if printVersion == true otherwise stderr